README.md

```diff
@@ -72,11 +73,10 @@ Looking forward to meet you,
 
 the devroom organizers:
 
-* Michael C. Jaeger, Fossology maintainer
-* Maximilian Huber, Fossology and SW360 committer
 * Philippe Ombredanne, ScanCode and AboutCode maintainer
 * Kate Stewart, SPDX and ACT @ Linux Foundation
-
+* Michael C. Jaeger, FOSSology maintainer, SW360 ...
```

Comment:
FWIW, I was using the alphabetical order, I am fine being first otherwise.

Reply:
OK